core_contracts/daofund/daofund.py

```diff
@@ -101,6 +104,15 @@ def addSymbolToSetdb(self) -> None:
         for symbol in assets:
             self._symbol.add(symbol)
 
+    @external
+    @only_owner
+    def addAddressToSetdb(self) -> None:
+        for symbol in self._symbol.range(0, len(self._address)):
```

Comment:
Doesn't this need to be len(self._symbol)? self._address will start at len = 0.

Reply:
yes